packages/gafl-webapp-service/src/__mocks__/mock-journeys.js

```diff
@@ -0,0 +1,346 @@
+        },
+        referenceNumber: '01300420-1WS0SGW-78CGFU',
+        endDate: '2020-04-29T23:00:00.000Z'
+      }
+    ],
+    cost: 54
+  },
+  finalisationResponse: {
+    payment: {
+      amount: 54,
+      timestamp: '2020-04-30T09:16:59.357Z',
+      source: 'Gov Pay',
+      method: 'Other'
+    }
+  },
+  setup: async () => {
+    await injectWithCookie('GET', NEW_TRANSACTION.uri)
+    await injectWithCookie('GET', CONTROLLER.uri)
+    await postRedirectGet(LICENCE_LENGTH.uri, { 'licence-length': '12M' })
+    await postRedirectGet(LICENCE_TYPE.uri, { 'licence-type': 'salmon-and-sea-trout' })
+    await postRedirectGet(LICENCE_TO_START.uri, { 'licence-to-start': 'after-payment' })
+    await postRedirectGet(DATE_OF_BIRTH.uri, dobHelper(dobSeniorToday))
+
+    fetch
+      .mockImplementationOnce(async () => new Promise(resolve => resolve({ json: () => mockPermits, ok: true })))
+      .mockImplementationOnce(async () => new Promise(resolve => resolve({ json: () => mockPermitsConcessions, ok: true })))
+      .mockImplementationOnce(async () => new Promise(resolve => resolve({ json: () => mockConcessions, ok: true })))
+
+    await injectWithCookie('GET', LICENCE_SUMMARY.uri)
+    await postRedirectGet(LICENCE_SUMMARY.uri)
+    await postRedirectGet(NAME.uri, { 'last-name': 'Graham', 'first-name': 'Willis' })
+    await postRedirectGet(ADDRESS_ENTRY.uri, goodAddress)
+    await postRedirectGet(CONTACT.uri, { 'how-contacted': 'none' })
+
+    fetch.mockImplementationOnce(async () => new Promise(resolve => resolve({ json: () => mockDefraCountries, ok: true })))
+    await injectWithCookie('GET', CONTACT_SUMMARY.uri)
+    await postRedirectGet(CONTACT_SUMMARY.uri)
+    await postRedirectGet(TERMS_AND_CONDITIONS.uri, { agree: 'yes' })
+
+    fetch.mockReset()
+  }
+}
+
+const JUNIOR_12_MONTH_LICENCE = {
+  transActionResponse: {
+    id: '8793ff10-6372-4e9c-b4f8-d0cde0ed2277',
+    expires: 1588412128,
+    dataSource: 'Web Sales',
+    permissions: [
+      {
+        permitId: 'd91b34a0-0c66-e611-80dc-c4346bad0190',
+        licensee: {
+          birthDate: '2006-01-01',
+          firstName: 'Graham',
+          lastName: 'Willis',
+          premises: '14 Howecroft Court',
+          street: 'Eastmead Lane',
+          town: 'Bristol',
+          postcode: 'BS9 1HJ',
+          country: 'United Kingdom',
+          preferredMethodOfNewsletter: 'Prefer not to be contacted',
+          preferredMethodOfConfirmation: 'Prefer not to be contacted',
+          preferredMethodOfReminder: 'Prefer not to be contacted'
+        },
+        issueDate: '2020-04-30T09:35:28.014Z',
+        startDate: '2020-04-29T23:00:00.000Z',
+        concession: {
+          concessionId: '3230c68f-ef65-e611-80dc-c4346bad4004',
+          proof: {
+            type: 'No proof'
+          }
+        },
+        referenceNumber: '01300420-1WS0JGW-FLNA84',
+        endDate: '2020-04-29T23:00:00.000Z'
+      }
+    ],
+    cost: 0
+  },
+  finalisationResponse: {
```

Comment:
Not a major issue but this is the request structure not the response stucture.

The structure here is not quite correct, its missing:
```

payment: {

}
```